packages/react/src/components/Notification/Notification.js

```diff
@@ -504,7 +505,12 @@ export class InlineNotification extends Component {
     );
 
     return (
-      <div {...other} role={role} kind={kind} className={classes}>
+      <div
+        {...other}
+        role={role}
+        kind={kind}
+        className={classes}
+        style={{ paddingRight: hideCloseButton ? spacing03 : '0' }}>
```

Comment:
Any reason you didn't do this via Sass code...?

Reply:
Hey @asudoh -- with the current structure of this component, the action button is a sibling to the close button & therefore it's difficult to target & change the action button when the close button is no longer present.

Adding a class is definitely an alternative approach. I just went with the solution that (seemed to) add/remove the least. Would you prefer to add a class and keep the padding update in Sass only?

Reply:
Thank you for sharing your thoughts @jendowns! I'm thinking about the ability of style override by applications, which tends to lean toward Sass approach. Is it changing `className` upon `hideCloseButton` enough to tell the Sass code that the close button is no longer present? If my words here is not clear enough please don't hesitate to speak up - Thanks!

Reply:
That's a good point @asudoh -- I just committed a change that goes the classname + Sass route. Let me know what you think -- thank you!